### PR TITLE
NET-502

### PR DESCRIPTION
**Pull request by uGiFarukh**
* Updated the docs to reflect latest netclient changes.

### CODE REVIEW

File: advanced-client-install.rst [hunk 1]
```diff
@@ -33,12 +33,12 @@ Windows will by default have firewall rules that prevent inbound connections. If
 
 ``netsh advfirewall firewall add rule name="Allow from <peer private addr>" dir=in action=allow protocol=ANY remoteip=<peer private addr>``
 
-If you want to allow all peers access, but do not want to configure firewall rules for all peers, you can configure access for one peer, and set it as a Relay Server.
+If you want to allow all peers access, but do not want to configure firewall rules for all peers, you can configure access for one peer, and set it as a Relay Server from Netmaker GUI.
```
Comment (theguy951357) [inline advanced-client-install.rst:36]: point out that relays are a pro feature and they'll need a self-hosted server to accomplish this.

**Review by theguy951357 — CHANGES_REQUESTED** [review]

File: advanced-client-install.rst (hunk 1 above)
Comment (uGiFarukh) [reply advanced-client-install.rst]: Pushed a commit with amendments.

**Review by theguy951357 — APPROVED** [review]

File: advanced-client-install.rst [hunk 2]
```diff
@@ -33,12 +33,12 @@ Windows will by default have firewall rules that prevent inbound connections. If
 
 ``netsh advfirewall firewall add rule name="Allow from <peer private addr>" dir=in action=allow protocol=ANY remoteip=<peer private addr>``
 
-If you want to allow all peers access, but do not want to configure firewall rules for all peers, you can configure access for one peer, and set it as a Relay Server.
+If you want to allow all peers access, but do not want to configure firewall rules for all peers, you can configure access for one peer, and set it as a Relay Server(Enterprise Edition Feature) from Netmaker GUI. In order to achieve this, self-hosted netmaker enterprise edition server is required.
```
Comment (abhishek9686) [inline advanced-client-install.rst:36]: It doesn't have to be self-hosted, a pro version can be created on SaaS as well
Change `Enterprise` terminology to `Professional`

File: advanced-client-install.rst [hunk 3]
```diff
@@ -151,17 +151,21 @@ Installation
 ======================
 
 
-To install netmaker, you need a server token for a particular network, unless you're joining a network that allows manual signup, in which case you can join without a token, but the server will quarantine the machine until the admin approves it.
+To install netmaker, you need an enrollment key for a particular network, unless you're joining a network that allows manual signup, in which case you can join without an enrolment key, but the server will quarantine the machine until the admin approves it.
```
Comment (abhishek9686) [inline advanced-client-install.rst:154]: To install netmaker?

Also we don't have any functionality of admin approval on server

File: advanced-client-install.rst [hunk 4]
```diff
@@ -171,10 +175,10 @@ Connect / Disconnect
 ----------------------
 
 **to disconnect from a network previously joined (without leaving the network):**
-  ``netclient disconnect -n <net name>``
+  ``netclient disconnect <netmaker network name>``
```
Comment (abhishek9686) [inline advanced-client-install.rst:178]: you can just say `network`

File: advanced-client-install.rst
Comment (abhishek9686) [inline advanced-client-install.rst:1]: also remove this section https://docs.netmaker.io/advanced-client-install.html#config-file-reference

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: advanced-client-install.rst (hunk 2 above)
Comment (uGiFarukh) [reply advanced-client-install.rst]: Pushed a commit with amendments.

File: advanced-client-install.rst (hunk 3 above)
Comment (uGiFarukh) [reply advanced-client-install.rst]: Pushed a commit with amendments.

**Pull request merged**